index.html

```diff
@@ -1,407 +1,433 @@
+                  <img src="assets/images/icons/Ellipse 1-2.png" 
+                    alt="Javascript" class="grid-item-icon"/>
+                  <label class="grid-item-text">CSS</label>
                 </div>
               </div>
-            </li>
-            <li class="stack-area">
-              <div>
-                <div class="stack-btn">
-                  <h5 class="stack-title">Skills</h5>
-                  <img
-                    src="assets/images/icons/arrowright.png"
-                    alt="dropdown icon"
-                    class="stack-img"
-                  />
+            </div>
+          </li>
+          <li class="stack-area">
+            <div>
+              <div class="stack-btn">
+                <h5 class="stack-title">Frameworks</h5>
+                <img src="assets/images/icons/arrowright.png" 
+                  alt="dropdown icon" class="stack-img"/>
+              </div>
+              <div class="hidden">
+                <div class="grid-container">
                 </div>
-                <div class="hidden">
-                  <div class="grid-container"></div>
+              </div>
+            </div>
+          </li>
+          <li class="stack-area">
+            <div>
+              <div class="stack-btn">
+                <h5 class="stack-title">Skills</h5>
+                <img src="assets/images/icons/arrowright.png" 
+                  alt="dropdown icon" class="stack-img"/>
+              </div>
+              <div class="hidden">
+                <div class="grid-container">
                 </div>
               </div>
-            </li>
-          </ul>
-        </div>
-      </section>
-    </main>
-    <footer></footer>
-  </body>
-</html>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="contact-section">
+      <div class="contact-background">
+        <form action="https://formspree.io/f/xbjevpkn" method="post"
+          class="contact-form">
+          <h2>Contact me</h2>
+          <p class="contact-form-paragraph">
+            If you have an application you are interested in developing, 
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it!
+          </p>
+          <div class="input-group">
+            <input type="text" name="name" id="name"
+              required maxlength="30"
+              placeholder="Write your name here">
+            <input type="email" name="email" id="email"
+              required
+              placeholder="Write your email here">
+            <textarea name="message" id="message"
+              required maxlength="500"
+              placeholder="Write your message here"></textarea>
+          </div>
+          <div class="form-footer">
+            <button class="form-submit-button" 
+              type="submit">Get in touch</button>
+          </div>
+        </form>
+      </div>
+    </section>
+  </main>
+  <footer></footer>
```

Comment:
Please make sure your design is the same as the given Figma design. You should stick to the Figma design according to the project requirements. Currently, I can't see the footer of the portfolio. KIndly, implement it.

![Screenshot from 2023-06-08 17-59-19](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/a3b8c684-d361-4167-b12b-668b833c1d23)

![image](https://github.com/chudisoft/Portfolio-Microverse/assets/101523582/084b0233-1a30-4525-81f8-7214b89ca705)